recipes/lib/get_opts.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import argcomplete
+
+from recipes.lib.utils import get_script_version
+
+
+script_list = []
+
+script_version_list = get_script_version()
+for script in script_version_list:
+    script = script.split(',')[0]
+    if script.endswith('.json'):
+        script_list.append(('.'.join(script.split('.')[:-1])).replace('_', '-'))
+
+parser = argparse.ArgumentParser(prog='retriever-recipes')
+
+# ..............................................................
+# Subparsers
+# ..............................................................
+subparsers = parser.add_subparsers(help='sub-command help', dest='command')
+new_json_parser = subparsers.add_parser('new_json', help='CLI to create retriever datapackage.json script')
+edit_json_parser = subparsers.add_parser('edit_json', help='CLI to edit retriever datapackage.json script')
+delete_json_parser = subparsers.add_parser('delete_json', help='CLI to remove retriever datapackage.json script')
+help_parser = subparsers.add_parser('help', help='')
+
+# ..............................................................
+# Subparsers With Arguments
+# ..............................................................
+edit_json_parser.add_argument('dataset', help='dataset name', choices=script_list)
+delete_json_parser.add_argument('dataset', help='dataset name', choices=script_list)
```

Comment:
I'm having a problem with the script_list where if I'm in another folder (even within retriever-recipes), the choices will show up blank. 

![image](https://user-images.githubusercontent.com/15080652/60351997-eeac7580-9994-11e9-9ac7-e0aec1dda250.png)

Reply:
@zhangcandrew  yes, currently it looks for only the `scripts` folder in the current directory. I am not sure, how we can always show the choices list when in any other directory.

Reply:
hmm, might be worth looking at what retriever does, I think if you play around with it you should be able to get it up and going :D